lectures/geom_series.md

```diff
@@ -856,12 +856,22 @@ so $\frac{\partial p_0}{\partial r}$ will always be negative.
 Similarly, $\frac{\partial p_0}{\partial g}>0$ as long as $r>g$, $r>0$ and $g>0$ and $x_0$ is positive, so $\frac{\partial p_0}{\partial g}$
 will always be positive.
 
-## Back to the Keynesian multiplier
+## Exercise
```

Comment:
@SylviaZhaooo is this at the end of the lecture? I think we should label this `Exercises` as a placeholder for any exercises added to the lecture.

Reply:
It is at the end of the lecture and I revised the title.

Thanks @mmcky for reviewing and I finished all the suggestions in #401.